pages/preprints.py

```diff
@@ -68,6 +68,14 @@ class PreprintSubmitPage(BasePreprintPage):
     upload_select_file = Locator(By.CSS_SELECTOR, '.file-browser-item > a:nth-child(2)')
     upload_file_save_continue = Locator(By.CSS_SELECTOR, 'div[class="p-t-xs pull-right"] > button[class="btn btn-primary"]')
 
+    # Author Assertions
+    public_available_button = Locator(By.ID, 'hasDataLinksAvailable', settings.QUICK_TIMEOUT)
+    public_data_input = Locator(By.CSS_SELECTOR, '.text-input-row .form-control')
```

Comment:
This one is tricky.  `.text-input-row .form-control` is very generic.  If we update the test to do Preregistration == "Available", there are now two things just in in `#author-assertions` that match that.

Reply:
Eh, for now let's just make it more precise by adding `#author-assertions` at the beginning of the css selector. It could still match the prereg available input, but we might need to live with that for now.  Maybe add a code comment saying "FRAGILE: needs to be updated if we start testing Preregistration is Available logic".

Reply:
Followup from offline discussion: we may be able to get around this with some clever (re:complicated) xpath expressions.